.gitignore

```diff
@@ -46,7 +46,7 @@ coverage.xml
 
 # Django stuff:
 *.log
-settings*.py
+# settings*.py
```

Comment:
@Emmanuel-Dominic why is commented out line still here?

Reply:
- Ensure resolved